### DISCUSSION TITLE
Lightbulb icon misaligned

### DISCUSSION THREAD

**Discussion by molnard** [community/community#3322] [Discussions]
Hi there,

Basically, the title says it all - the lightbulb icon seems to be smaller than the other icon those are standard weight sizes. 

![image](https://user-images.githubusercontent.com/9844978/107012275-1c74b580-6799-11eb-86fa-83b9c0e5784e.png)

As a result, the categories list in our discussion board is a bit bumpy.

**Comment by PranamBhat** [comment]
Oops!! You're doing damn cool testing! 

GitHub :  I'm done!
@molnard : Wait, let me find a bug!!!
